site/blog/2022/deploy-static-website.mdx

```diff
@@ -0,0 +1,13 @@
+---
+slug: deploy-static-website
+title: Deploy a Static Website Using SQL
+date: 2022-12-21
+unlisted: true
```

Comment:
I don't think this works. You can ignore it in the docusaurus config if you would like though. We do it for the clean aws account one